### ISSUE TITLE
[Enhancement] Teleport to Slayer Master Items: Rada's Blessing 4 for Konar / Slayer Ring Eternal for Nieve

### ISSUE THREAD

**Issue by Rallik-Pog** [PTYB/SlayerIssueTracker#214]
Think it would be a good feature to add to teleport directly to the master via items.

Dax handles this sometimes, but has been inconsistent using the items.

**Comment by PTYB** [comment]
It only supports POH, are those POH items?

**Comment by Rallik-Pog** [comment]
No, they are both items. Dax sometimes will take the long route instead of using a slayer ring eternal or rada's blessing 4 as a means of getting to the slayer master.

**Comment by PTYB** [comment]
If it does not use those teleports it would be a dax bug.

**Comment by Rallik-Pog** [comment]
Brought it up with Elias about Dax, even though the weight is 0 its still inconsistently using slayer ring to tasks and rada to master. It will end up walking there half the time, even though its equipped or in inventory.
